packages/core/src/components/portal/portal.tsx

```diff
@@ -49,6 +54,9 @@ const REACT_CONTEXT_TYPES: ValidationMap<IPortalContext> = {
 export class Portal extends React.Component<IPortalProps, IPortalState> {
     public static displayName = `${DISPLAYNAME_PREFIX}.Portal`;
     public static contextTypes = REACT_CONTEXT_TYPES;
+    public static defaultProps: IPortalProps = {
+        container: typeof document !== "undefined" ? document.body : null,
```

Comment:
`container` can possibly be null so you'll need to check for that in `componentDidMount` and possibly fail.